src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.hiqes.andele" >
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.hiqes.andele">
+
+    <application
+        android:appComponentFactory="android.support.v4.app.CoreComponentFactory"
```

Comment:
Why is this necessary?  What implication will this have for applications which use this library?